src/main/java/org/folio/entlinks/domain/entity/SourceEnum.java

```diff
@@ -0,0 +1,17 @@
+package org.folio.entlinks.domain.entity;
+
+public enum SourceEnum {
```

Comment:
I think you can choose more expressive name (without ```...Enum``` as our other enums), some alternatives: ```AuthoritySource```, ```AuthorityFileSource```, ```AuthoritySourceType```